api/python/cellxgene_census/src/cellxgene_census/experimental/pp/_stats.py

```diff
@@ -46,6 +47,10 @@ def mean_variance(
         ddof:
             "Delta Degrees of Freedom": the divisor used in the calculation for variance is ``N - ddof``, where ``N`` represents the number of elements.
 
+        exclude_zeros:
```

Comment:
In addition, with the decision to define `N` as `nnz` when this flag is True, I think we should rename it.  It is no longer `exclude zeros`, but rather `use nnz as N`.

My suggestion is to rename for clarity - possible candidates:
* nnz_only
* use_nnz_as_sample_count

etc.  @pablo-gar - do you have a good name that would be clear and descriptive?  `exclude_zeros` seems misleading.

Reply:
I agree. Let's wait for Pablo to chime in.

Reply:
For the time being, I settled for `nnz_only`.

Reply:
As long as @pablo-gar is good with this change, I'm good....